stages/org.osbuild.ovf

```diff
@@ -116,8 +117,9 @@ def write_manifest(vmdk, ovf):
 
 
 def main(options, tree):
+    ostype = options["ostype"]
```

Comment:
This should be `options.get("ostype", "other26xLinux64Guest")` since it's optional and has a default.

Reply:
Fixed